legacy-sdk/whirlpool/tests/utils/init-utils.ts

```diff
@@ -156,10 +156,15 @@ export async function buildTestAquariums(
 
   const aquariums: TestAquarium[] = [];
   // Airdrop SOL into provider wallet;
-  await ctx.connection.requestAirdrop(
+  const balance = await ctx.connection.getBalance(
```

Comment:
i recall seeing this at least twice in this commit. do you think it's worth adding a utility here that's like `airdropIfBelowThreshold` with an (optional) confirmation level.. naming could probably be improved though